src/devSVG.cpp

```diff
@@ -418,6 +457,13 @@ void svg_text(double x, double y, const char *str, double rot,
               double hadj, const pGEcontext gc, pDevDesc dd) {
   SVGDesc *svgd = (SVGDesc*) dd->deviceSpecific;
 
+  // If we specify the clip path inside <text>, the "transform" also
+  // affects the clip path, so we need to specify clip path at an outer level
+  if(svgd->clipno > 0) {
```

Comment:
All these `if` need a space (`if (`) to be consistent with the rest of the style